gui/default/assets/lang/lang-en.json

```diff
@@ -281,6 +289,7 @@
    "Start Browser": "Start Browser",
    "Statistics": "Statistics",
    "Stopped": "Stopped",
+   "Stores and syncs only encrypted data. Folders on all connected devices need to be set up with the same password or be of type \"{%receiveEncrypted%}\" too.": "Stores and syncs only encrypted data. Folders on all connected devices need to be set up with the same password or be of type \"{{receiveEncrypted}}\" too.",
```

Comment:
The `stores` part is a false statement. My local data is not encrypted.

I guess the right term is encrypts outgoing, decrypts incoming data

Reply:
This describes the receive-only folder type. It does receive encrypted data, not decrypt, and stores encrypted data.